packages/forms-web-app/src/views/final-comment/comments-question.njk

```diff
@@ -0,0 +1,61 @@
+{# todo: change layout based on appeal type when component reused #}
+{% extends "layouts/full-appeal-banner/main.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set title = "Do you want to submit final comment? - Appeal a planning decision - GOV.UK" %}
```

Comment:
It feels like this should read "Do you want to submit a final comment" but I can see it matches the copy in the ticket. Perhaps one to check with design?